script/Ciel/Day 14/14_01B_01_CIEL14B_KRAKE.txt

```diff
@@ -399,8 +399,8 @@ Then, as if to finish it off,
 [sha:7c50aab13014cb6a8b4cc18ffa1abdc88b506df6]{
 -- Page 33, Offset 38419.
 -- その外見通り、不死身と思われた怪物もここまで破壊されては退場する他ない。
-Just as looking at the extent of the destruction would lead you to believe,
- not even this seemingly immortal monster could come back after suffering such devastation.
+Just as looking at the extent of the destruction would lead you to believe
```

Comment:
Revert, this changes the meaning from
"As you might imagine after seeing how fucked up it is, it's dead" 
to something nearer
"Looking at it would lead you to believe it's dead [but it isn't]"
but also results in a sentence that feels like it's following something up, but isn't